dev-notes/minimal-os-for-fobnail.md

```diff
@@ -50,6 +50,7 @@ could be implemented by us.
 | Bootloader Capabilities | Required to boot target OS                                                     |
 | C library               | [Fobnail Attester](https://github.com/fobnail/fobnail-attester) is writen in C |
 | Bootable by SKL         | Whether OS can be loaded by TrenchBoot SKL without SKL or OS modification      |
+| License                 | Whether given OS is open sourced can be fully used on embedded devices         |
```

Comment:
@TomaszAIR this sound weird, maybe you mean:

```
Whether OS use open-source license, and if licese used is applicable for embedded market
```

Reply:
@pietrushnic I also did not like that description but I was not able to come up with something nicer. Thanks. Added here [0964cb8](https://github.com/fobnail/docs/pull/26/commits/0964cb82943ee643e07d5fe643620ff5d040275e)